installer/release.sh

```diff
@@ -27,23 +27,64 @@ version=$1
 cd "$(dirname "$0")" ||
 die "Could not switch directory"
 
+# Export paths to inno setup file
 SCRIPTDIR="$(pwd -W)"
 ROOTDIR="$(cd / && pwd -W)"
+export SCRIPTDIR ROOTDIR
+
+# Evaluate architecture
+ARCH="$(uname -m)"
+
+case "$ARCH" in
+i686)
+	BITNESS=32
+	;;
+x86_64)
+	BITNESS=64
+	;;
+*)
+	die "Unhandled architecture: $ARCH"
+	;;
+esac
 
 # Generate list of files to include
-printf > file-list.iss
-for f in gpl-2.0.rtf git.bmp gitsmall.bmp
+pacman_list () {
+	pacman -Ql $(for arg
+		do
+			pactree -u "$arg"
+		done |
+		sort |
+		uniq) |
+	grep -v '/$' |
+	sed 's/^[^ ]* //'
+}
+
+LIST="$(pacman_list mingw-w64-$ARCH-git git-extra ncurses mintty vim \
+	sed awk less grep gnupg findutils coreutils \
+	dos2unix which subversion |
+	grep -v -e '\.[acho]$' -e '/aclocal/' \
+		-e '/man/' \
+		-e '^/usr/include/' -e '^/mingw32/include/' \
+		-e '^/usr/share/doc/' -e '^/mingw32/share/doc/' \
+		-e '^/usr/share/info/' -e '^/mingw32/share/info/' |
+	sed 's/^\///')"
+
+LIST="$LIST etc/profile etc/bash.bash_logout etc/bash.bashrc etc/fstab"
+LIST="$LIST mingw$BITNESS/etc/gitconfig"
+
+rm -rf file-list.iss
+for f in $LIST
 do
-	printf 'Source: %s; DestDir: {app}; Flags: %s; AfterInstall: %s\n' \
-		$f replacesameversion DeleteFromVirtualStore \
-	>> file-list.iss
+	printf 'Source: %s; DestDir: {app}\%s; Flags: %s; AfterInstall: %s\n' \
+                $f $(dirname $f) replacesameversion DeleteFromVirtualStore \
```

Comment:
Yeah... I think we can convert that to a `sed` call relatively easily. Something like

```bash
echo "$LIST" |
sed -e 's|/|\\|g' \
  -e 's|^\([^\\]*\)$|Source: \1; DestDir: {app}; Flags: replacesameversion; AfterInstall: DeleteFromVirtualStore|' \
  -e 's|^\(.*\)\\\([^\\]*\)$|Source: \1\\\2; DestDir: {app}\\\1; Flags: replacesameversion; AfterInstall: DeleteFromVirtualStore|' \
  >> file-list.iss
```

(the first expression replaces forward with backward slashes, the second expression handles files in the root directory, the third expression handles files in subdirectories.)